static/src/javascripts-legacy/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -280,6 +280,14 @@ define([
         }
     };
 
+    function constructURL(base, params) {
```

Comment:
We actually already have a function to do this: https://github.com/guardian/frontend/blob/0dea0c4040d6e583dd9d780b28b92379dfcb0a5b/static/src/javascripts/lib/url.js#L49